From 5971b6afa801868d4bf790727dec254588832d70 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Sat, 3 Dec 2011 01:36:05 +0100
Subject: [PATCH] path: use %m instead of strerror(errno)

and strerror(-errno) was just wrong.
(cherry picked from commit 768147d13d0877a4c3e5f6f986c3064de62ff4f1)
---
 src/path.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/path.c b/src/path.c
index f15c921..142fd2d 100644
--- a/src/path.c
+++ b/src/path.c
@@ -556,7 +556,7 @@ static void path_fd_event(Unit *u, int fd, uint32_t events, Watch *w) {
         }
 
         if (ioctl(fd, FIONREAD, &l) < 0) {
-                log_error("FIONREAD failed: %s", strerror(errno));
+                log_error("FIONREAD failed: %m");
                 goto fail;
         }
 
@@ -568,7 +568,7 @@ static void path_fd_event(Unit *u, int fd, uint32_t events, Watch *w) {
         }
 
         if ((k = read(fd, buf, l)) < 0) {
-                log_error("Failed to read inotify event: %s", strerror(-errno));
+                log_error("Failed to read inotify event: %m");
                 goto fail;
         }
 
